### DISCUSSION TITLE
16 Bit Parallel for ILI9341

### DISCUSSION THREAD

**Discussion by dcinnovation-mt** [Bodmer/TFT_eSPI#1777] [New processor/display support or capability request]
Hello,

I am developing a project that uses a RP2040 and a ILI9341 configured to a 16 bit parallel MCU interface.

I was wondering if you would be interested in taking the development on as a contract, and if not if you are able to give me a high level overview of what I need to change to make it work?

Thanks in advance,
DC

**Comment by Bodmer** [comment]
Hello, email me bodmer(at)anola.net and let me know what your requirements are.

Adding 16 bit parallel write only using the PIO should be quite easy and would give fast screen updates. The library already supports 8 bit parallel with DMA on the RP2040 so that may provide the performance you need.
